src/vs/vscode.d.ts

```diff
@@ -6781,6 +6781,30 @@ declare module 'vscode' {
 		 * @param callback A command handler function with access to an [editor](#TextEditor) and an [edit](#TextEditorEdit).
 		 * @param thisArg The `this` context used when invoking the handler function.
 		 * @return Disposable which unregisters this command on disposal.
+		 *
+		 *
+		 * **Note:**  The `registerTextEditorCommand` should run synchronously, as it gets passed in a
```

Comment:
Notes arent written in bold but italic
```suggestion
		 * *Note:*  The `registerTextEditorCommand` should run synchronously...
```